### PR TITLE
More updates of expression help

### PR DESCRIPTION
**Pull request by DelazJ**
- Expose default values of args
- Fix args name or description
- Fix/add more examples (to show various possibilities of a function or so that functions that relate have examples that relate)

### CODE REVIEW

File: resources/function_help/json/to_dm [hunk 1]
```diff
@@ -7,10 +7,11 @@
     {"arg":"coordinate","description":"A latitude or longitude value."},
     {"arg":"axis","description":"The axis of the coordinate. Either 'x' or 'y'."},
     {"arg":"precision", "description":"Number of decimals."},
-    {"arg":"formatting", "optional": true, "default":"", "description":"Designates the formatting type. Acceptable values are NULL, 'aligned' or 'suffix'."}
+    {"arg":"formatting", "optional": true, "default":"", "description":"Designates the formatting type. Acceptable value is NULL (default), 'aligned' or 'suffix'."}
```
Comment (nyalldawson) [inline resources/function_help/json/to_dm:10]: This reads worse to me -- why the change?

Comment (DelazJ) [reply resources/function_help/json/to_dm:10]: - the `"default" : ""` does not render in QGIS, hence the precision next to the NULL value
- with a plural form, I'd expect an "AND" list so either `Acceptable values are NULL, 'aligned' and 'suffix'` or use the singular form with "OR".

Comment (nyalldawson) [reply resources/function_help/json/to_dm:10]: Mmm... it doesn't read well to me -- it's just awkward grammar. I'm not an English teacher, but I'm 99.9% sure the original form is correct.

Comment (DelazJ) [reply resources/function_help/json/to_dm]: Reverted...

**Review by nyalldawson — APPROVED** [review]
